src/_scss/site/_globalFooter.scss

```diff
@@ -2,6 +2,7 @@
 .usa-da-page-content {
     min-height: 100%;
     padding-bottom: 55px;
+    overflow: auto;
```

Comment:
What is this changing?

Reply:
It allows the page to scroll when the dropdown is fully extended( since its so close to the bottom of the page). Without the scrolling, the dropdown runs over the footer

Reply:
Got it- thanks!